.github/workflows/devnet.yaml

```diff
@@ -40,6 +46,13 @@ jobs:
             - name: Build
               run: pnpm build --filter @sunodo/devnet
 
+            - name: Publish
+              if: ${{ inputs.release }}
+              run: pnpm publish --access public
+              working-directory: packages/devnet
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
```

Comment:
Won't this conflict with https://github.com/sunodo/sunodo/pull/479/commits/04d3d1e7d17c9b822a29cfbf1ed123fad98b45cd#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR44 ?

Reply:
Why?

Reply:
Nevermind, I was thinking that it would be possible that the `publish` was going to be executed again.